packages/utils/logger/src/configs/index.ts

```diff
@@ -0,0 +1,4 @@
+import createDefaultConfiguration from './default-configuration';
+import createOutputFileConfiguration from './output-file-configuration';
+
+export { createDefaultConfiguration, createOutputFileConfiguration };
```

Comment:
Same as for the format exports, you could directly re-export the modules instead of importing them first